### DISCUSSION TITLE
Windows Lan Docker Access

### DISCUSSION THREAD

**Discussion by williamlabrum** [TriliumNext/Trilium#4101] [Q&A] [answered]
Trying to figure out how to access the docker container on my LAN @ home. The server is running in Docker container on Windows 10. 

Wiki mentioned setting up a network on docker but that was specific to macvlan. I've tried --net="host" for docker run command as well (no dice). Also updated windows firewall for port 8080 (still no dice). The only thing that works is local host access w/o --net arg . 

Any tips are greatly appreciated. (Kind of wish I was running Linux at this point but I feel too close to pivot now)

**Comment by meichthys** [comment]
TBH, It'll probably be worth it to pivot to Linux now rather than later.  
- [Install Linux](https://ubuntu.com/tutorials/install-ubuntu-desktop#1-overview) (Debian/Ubuntu is great for starting out since there is a LOT of community support and documentation out there).
- [Install Docker](https://docs.docker.com/engine/install/ubuntu/)
- [Run Trilium](https://github.com/zadam/trilium/wiki/Docker-server-installation) using [docker compose](https://docs.docker.com/compose/install/#scenario-two-install-the-compose-plugin)
- Use [Tailscale](https://tailscale.com/kb/1031/install-linux/) to vpn into your local network
- Connect to Trilium as you normally would

**Reply by williamlabrum** [reply, marked as answer]
Thanks for the suggestion. Seems it was just a windows firewall issue in the end. Computer was blocking ping 🙄 

After clearing up Windows firewall issues i ended up using the following on windows with docker
```
docker run -d -p <windows_ip>:8080:8080 -v ~/trilium-data:/home/node/trilium-data zadam/trilium:x.xx.x
```
Now i can access notes on internal network. 🥂
